core/src/main/java/org/apache/carbondata/core/scan/filter/executer/ImplicitIncludeFilterExecutorImpl.java

```diff
@@ -53,6 +53,14 @@ public BitSetGroup applyFilter(RawBlockletColumnChunks rawBlockletColumnChunks,
     return bitSetGroup;
   }
 
+  @Override
+  public BitSet prunePages(RawBlockletColumnChunks rawBlockletColumnChunks)
+      throws FilterUnsupportedException, IOException {
+    BitSet bitSet = new BitSet(rawBlockletColumnChunks.getDataBlock().numberOfPages());
+    bitSet.set(0, rawBlockletColumnChunks.getDataBlock().numberOfPages());
```

Comment:
add a local variable for `rawBlockletColumnChunks.getDataBlock().numberOfPages()`